Azure-Samples/modern-data-warehouse-dataops
# upload config file for data retention

# Type of PR
- Code changes
- CI-CD changes

## Purpose
Data Retention - add configuration file as part of the deployment steps

## Author pre-publish checklist

## Validation steps
- run the deploy.sh script and verify that the configuration file has been uploaded

## Issues Closed or Referenced

- Closes #581
